src/mixins/canvas_events.mixin.js

```diff
@@ -352,8 +352,8 @@
       this.fire('mouse:down', { e: e });
 
       var target = this.findTarget(e);
-      if (typeof target !== 'undefined') {
-        target.fire('mousedown', { e: e, target: target });
+      for (var i = 0; i < this.targets.length; i++) {
```

Comment:
this is the only way i could come out to manage one-click-many-targets events firing. Does it looks ok for you?